tests/Operation/DistinctTest.php

```diff
@@ -51,4 +53,28 @@ public function provideInvalidConstructorOptions()
 
         return $options;
     }
+
+    public function testExplainableCommandDocument(): void
+    {
+        $options = [
+            'collation' => ['locale' => 'fr'],
+            'maxTimeMS' => 100,
+            'readConcern' => new ReadConcern(ReadConcern::LOCAL),
+            'readPreference' => new ReadPreference(ReadPreference::SECONDARY_PREFERRED),
```

Comment:
Depending on the topology to which the driver is connected, this might result in a `$readPreference` option appearing in the outgoing command (see: [OP_MSG: Global Command Arguments](https://github.com/mongodb/specifications/blob/master/source/message/OP_MSG.rst#global-command-arguments)); however, I have no idea if that's relevant to `explain`. This may be another thing to ask the server query team about if you're so inclined.

Otherwise, lumping this towards the end of the `$options` array under `// Intentionally omitted options` (or something to that effect) seems good. Similar to what you'd do for Explainable operations that take a `writeConcern` option.